### ISSUE TITLE
Composer v2 compatibility

### ISSUE THREAD

**Issue by manavo** [typesense/typesense-php#1]
## Description

The library is not fully PSR-4 compatible, and will cause an issue with Composer v2.

## Steps to reproduce

Just running the `composer require` command, and the warning comes up.

## Expected Behavior

No warning to come up.

## Actual Behavior

There's a warning:

```
Deprecation Notice: Class Typesense\Lib\Configuration located in ./vendor/typesense/typesense-php/src/lib/Configuration.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Typesense\Lib\Node located in ./vendor/typesense/typesense-php/src/lib/Node.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by AbdullahFaqeir** [comment]
Hi @manavo, 

We'll work on fixing it ASAP.

Thanks for bringing this to attention.

Cheers.

**Comment by AbdullahFaqeir** [comment]
Fix with release #v4.0.1.

**Comment by manavo** [comment]
Thanks for the super quick turnaround!
